viewmaster/viewmaster.py

```
@@ -2199,6 +2494,10 @@ def trim_html(html):
 ################################################################################
 
 if __name__ == "__main__":
+    logger = create_logger()
+    pdsviewable.load_icons(path=ICON_ROOT_, url=ICON_URL_, color=ICON_COLOR,
+                           logger=logger)
+    initialize_caches(reset=False)
     app.run(host='0.0.0.0', port=8080, debug=True)
```

Ruff (0.14.10)

Possible binding to all interfaces

(S104)